controllers/note.controllers.js

```diff
@@ -0,0 +1,27 @@
+const note = require('../models/note.model');
+const customError = require('../models/CustomError');
+const mongodb = require("../db/client.db");
+
+
+exports.create = async (req, res) => {
+
+    //Si pas de body
+    if (!req.body || !req.body.content) {
+        return res.send(new customError("Le contenu est requis", 401));
+    }
+
+    const userID = req.locals.userID;
+    const content = req.body.content;
+
+    const newNote = note.create(userID, content);
```

Comment:
Il faut gérer les cas d'erreurs

Reply:
Si c'est pour la création de la note cela n'est pas nécessaire. 
Documentation publique de fastify:

Catching Uncaught Errors In Fastify
Fastify tries to catch as many uncaught errors as it can without hindering performance. This includes:

synchronous routes, e.g. app.get('/', () => { throw new Error('kaboom') })
async routes, e.g. app.get('/', async () => { throw new Error('kaboom') })
The error in both cases will be caught safely and routed to Fastify's default error handler for a generic 500 Internal Server Error response.

For customizing this behaviour, you should use setErrorHandler

Reply:
Le truc c'est qu'il ne faut pas renvoyer une erreur générique 500 mais l'erreur que l'on a créé nous en faite...

Reply:
J'ai survolé le reste de la doc, tu as raison 😄 
Je vais implémenter la méthode qui gère les erreurs.
Ref : https://www.fastify.io/docs/latest/Server/#seterrorhandler